index.html

```diff
@@ -4,26 +4,29 @@
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Microverse Portfolio</title>
-  <link rel="stylsheet" href="style.css">
+  <title>Microverse portfolio</title>
+  <link rel= "stylesheet" href="style.css"/>
 </head>
 
 <body>
-  <header class="header">
-    <nav class="menu">☰ </nav>
-  </header>
-  <main>
-    <section class="Home">
-      <h1>Sonia Bangar</h1>
-      <p class="paragraph">Hello! I am a software developer! I can help you build a product, feature or website. Taka a
-        look of my works.don&apos;t
-        If you like what you see and have a project you need coded, hesitate and contact me.</p>
-      <button class="start">Start collaboration</button>
-
+    <div class= "toolbar">
+      <div class= "img_geomtry">
+        <img src= "./images/image_geometry_1.png" alt=""class="shape1">
+      </div>
+      <nav class= "navbar">
+        <img class= "menu" src= "./images/ic_menu.png" alt="">
+      </nav>
+    </div>
+    <section class= "header">
+      <h1 class= "title">Sonia Bangar</h1> 
+      <p class= "paragraph">
+        Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+If you like what you see and have a project you need coded, don't hesitate and contact me.
+      </p>
+      <button class="bttn">Start collaboration</button>
+      <img class="spacer" src="/images/image_geometry_2.svg" alt="spacer">
```

Comment:
- [ ] Please make sure that The section has the proper background image (decorations are always background images, not `img` tags). If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.